app/controllers/articles/index.js

```diff
@@ -1,11 +1,16 @@
 import Ember from 'ember';
 
 export default Ember.ArrayController.extend({
-  contentDidChange: function () {
-    console.log('Called when we add or removed an article');
-  }.observes('model.[]'),
+  queryParams: ['showReturned'],
+  showReturned: false,
 
-  stateDidChange: function () {
-    console.log('Called when the state prop changes for any article');
-  }.observes('model.@each.state')
+  filteredResults: function () {
```

Comment:
This is hosing things. Just by having this filterResults be here and using it to render the articles in the template we have trouble. The articles will for the first friend loaded "stick". If you go to a friend, then back to friends index, then to another friend. Their articles will be the first friend's articles. The filterResults will only change to a new friends on page reload.